src/screens/AdminPortal/OrganizationFunds/modal/FundModal.tsx

```
@@ -182,103 +150,110 @@ const FundModal: React.FC<InterfaceFundModal> = ({
+          </FormFieldGroup>
+        </Form.Group>
+
+        <div
+          className={`d-flex mt-2 mb-3 flex-wrap ${
+            mode === 'edit'
+              ? 'justify-content-between'
+              : 'justify-content-start gap-3'
+          }`}
+        >
+          <Form.Group className="d-flex">
+            <label>{t('taxDeductible')}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)